lib/author.rb

```diff
@@ -0,0 +1,28 @@
+class Author
+  attr_reader :first_name, :last_name, :books
```

Comment:
be careful not to expose things like first/last names here when we don't ask for this